### PR TITLE
fix: resolve implementation for multiple contributors

### PR DESCRIPTION
**Pull request by tenshiAMD**
**What**:

- resolve implementation for multiple contributors

**Why**:

- multiple contributors does not work properly

**How**:

- refactor and update implementation

**Checklist**:

- [x] Documentation
- [ ] Ready to be merged 
- [x] Added myself to contributors table. 
[Bot Usage](https://allcontributors.org/docs/en/bot/installation#4-update-your-contributing-documentation)

### CODE REVIEW

**Comment by tenshiAMD** [comment]
TODO:

- [ ] Add snapshot/s for testing multiple contributors in single comment

File: test/integration/issue_comment.test.js [hunk 1]
```diff
@@ -761,4 +765,86 @@ describe("issue_comment event", () => {
...
+
+    await probot.receive({
+      name: "issue_comment",
+      id: "1",
+      payload: issueCommentCreatedMultipleContributorsPayload,
+    });
+
+    expect(mock.activeMocks()).toStrictEqual([]);
+    expect(output).toMatchSnapshot("logs");
+  });
```
Comment (tenshiAMD) [inline test/integration/issue_comment.test.js:849]: @gr2m @Berkmann18 please check this one as I cannot find any solution for this one to make it somehow pass. I need your advice regarding this matter. Thank you.

Comment (gr2m) [reply test/integration/issue_comment.test.js:849]: I can probably look into it next week

Comment (tenshiAMD) [reply test/integration/issue_comment.test.js:849]: > I can probably look into it next week

@gr2m please do check thanks

Comment (gr2m) [reply test/integration/issue_comment.test.js:849]: When you see a "Nock: No match for request" error, a good way to debug it is to run the test with nock debugging enabled

```
DEBUG=nock* npx jest --testNamePattern "Happy path, add correct new multiple contributors" test/integration/issue_comment.test.js
```

It seems that the app attempts to send these requests twice:

- `GET /repos/all-contributors/all-contributors-bot/contents/.all-contributorsrc?ref=master`
- `GET /repos/all-contributors/all-contributors-bot/contents/README.md?ref=master`
- `GET /repos/all-contributors/all-contributors-bot/git/ref/heads%2Fmaster`

nock supports mock the same requests twice 

```js
      .get(
        "/repos/all-contributors/all-contributors-bot/contents/.all-contributorsrc?ref=master"
      )
      .twice()
      .reply(200, reposGetContentsAllContributorsRCdata)
```

however this looks like a bug?

Comment (tenshiAMD) [reply test/integration/issue_comment.test.js:849]: @gr2m I already tried `DEBUG=nock*` and did have some other details but still not helpful 😞 

> however this looks like a bug?

Looks like it is. I also tried updating `nock` to the latest version just now but it still does not work.

Comment (gr2m) [reply test/integration/issue_comment.test.js:849]: I mean it looks like a bug with the code, not with nock. Why is the code sending these requests twice? Does it send them once per contributor?

Comment (tenshiAMD) [reply test/integration/issue_comment.test.js:849]: > Does it send them once per contributor?

@gr2m yes currently it is one PR per contributor. Making one PR for many contributors is still on our TODO list for improvements.

Comment (tenshiAMD) [reply test/integration/issue_comment.test.js]: @gr2m looks like all is good now. anything else we missed?

**Review by gr2m — APPROVED** [review]
# 🚀:shipit:

**Pull request merged**